common/services/reference-data.js

```diff
@@ -83,13 +84,18 @@ const referenceDataService = {
     })
   },
 
-  getLocationsBySupplierId(supplierId) {
-    return referenceDataService.getLocations({
-      filter: {
-        cache: false,
-        'filter[supplier_id]': supplierId,
-      },
+  async getLocationsBySupplierId(supplierId) {
```

Comment:
Is there any reason other than the different URL as to why this needs to be handled by a different "REST" client?

After our conversation yesterday and us already having a model for this, it would seem more sensible for the API to only maintain one endpoint that does this action. This appears to have introduced a duplicate endpoint with similar functionality.

Reply:
> Is there any reason other than the different URL as to why this needs to be handled by a different "REST" client?

No

> After our conversation yesterday and us already having a model for this, it would seem more sensible for the API to only maintain one endpoint that does this action. This appears to have introduced a duplicate endpoint with similar functionality
Yes.